src/main/java/bio/terra/service/filedata/flight/ingest/IngestFilePrimaryDataLocationStep.java

```diff
@@ -38,6 +38,7 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
                 GoogleBucketResource bucketForFile =
                     resourceService.getOrCreateBucketForFile(
                         dataset,
+                        dataset.getId().toString(),
```

Comment:
If this argument is always `dataset.getId()` then it can be done inside `getOrCreateBucketForFile()` instead of having the caller do it.